applications/GeoMechanicsApplication/tests/cpp_tests/test_newmark_quasistatic_U_Pw_scheme.cpp

```diff
@@ -0,0 +1,126 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "testing/testing.h"
+
+#include "custom_strategies/schemes/newmark_quasistatic_U_Pw_scheme.hpp"
+#include "spaces/ublas_space.h"
+#include "test_utilities/spy_element.h"
+#include "test_utilities/spy_condition.h"
+
+namespace Kratos::Testing {
+
+using namespace Kratos;
+using SparseSpaceType = UblasSpace<double, CompressedMatrix, Vector>;
+using LocalSpaceType = UblasSpace<double, Matrix, Vector>;
+
+class NewmarkQuasistaticUPwSchemeTester {
+public:
+    Model mModel;
+    NewmarkQuasistaticUPwScheme<SparseSpaceType, LocalSpaceType> mScheme =
+        CreateValidScheme();
+
+    NewmarkQuasistaticUPwSchemeTester()
+    {
+        CreateValidModelPart();
+    }
+
+    NewmarkQuasistaticUPwScheme<SparseSpaceType, LocalSpaceType> CreateValidScheme() const
+    {
+        NewmarkQuasistaticUPwScheme<SparseSpaceType, LocalSpaceType> result(0.25, 0.5, 0.75);
+        return result;
```

Comment:
I would eliminate the redundant variable:
```suggestion
        return NewmarkQuasistaticUPwScheme<SparseSpaceType, LocalSpaceType> result{0.25, 0.5, 0.75};
```

Reply:
Done